packages/node-core/src/integrations/http/SentryHttpInstrumentation.ts

```diff
@@ -28,12 +29,22 @@ export type SentryHttpInstrumentationOptions = InstrumentationConfig & {
    */
   breadcrumbs?: boolean;
 
+  /**
+   * Whether to create spans for requests or not.
+   * As of now, creates spans for incoming requests, but not outgoing requests.
+   *
+   * @default `true`
```

Comment:
l: Likely makes sense to mention this here (assuming it still applies?)

```suggestion
   * @default `true` unless `skipOpenTelemetrySetup` is set to `true` in `Sentry.init` options
```